sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/azure.yaml

```diff
@@ -200,6 +213,12 @@ provider:
       metadata:
         name: Azure subscription keys should not be leaked
       detection:
+        pre:
+          include:
+            content:
```

Comment:
The `content` pre-check is an AND list, correct? If so, we might miss some things by filtering on both "secret" and "key", if the pattern only requires one of them.

Reply:
No, I think that's an OR, but I'm gonna ask Jonas

Reply:
it's an OR, he confirmed

Reply:
In that case, I would not filter on "secret" or "key" since they will be more generic than "microsoft.com"

Reply:
totally yeah actually